text/rocq-roadmap.md

```diff
@@ -5,14 +5,14 @@ Rocq Long Term Roadmap
 
 The Rocq Development Team, building on 40 years of experience with Coq, is pleased to announce the Rocq Prover, the next-generation proof assistant. The Rocq Prover is designed to empower a diverse range of users, from mathematicians seeking formal rigor to software engineers building high-assurance systems.
 
-While Rocq leverages the rich heritage of Coq, it embraces a more agile development philosophy. This allows us to deliver impactful changes more frequently, accelerating innovation and user experience improvements.
+While Rocq leverages the rich heritage of Coq, it aims to keep evolving: deliver impactful changes, enable new research and improve the user experience.
 
 This roadmap outlines our vision for Rocq's future, focusing on four key pillars:
 
 + **Open and Accessible**: The Rocq Prover provides user-friendly installation, comprehensive educational resources, and a thriving community to ensure everyone can benefit from its power.
-+ **Trustworthy Proofs**: The Rocq Prover prioritizes self-verification and performance optimization, ensuring the highest level of trust and efficiency in your formal proofs.
-+ **Effortless Maintainability**: We are committed to a clean and streamlined codebase, simplifying future development and reducing the burden on users.
-+ **Enhanced Usability**: We will explore AI-powered features, streamlined domain-specific tools, and do our best to improve day-to-day usability for mathematicians, educators, and engineers alike.
++ **Trustworthy**: The Rocq Prover prioritizes self-verification and performance optimization, ensuring the highest level of trust and efficiency in your formal proofs.
```

Comment:
```suggestion
+ **Trustworthy**: The Rocq Prover prioritizes self-verification and high performance, ensuring the highest level of trust and efficiency in your formal proofs.
```

"The Rocq Prover prioritizes self-verification" - not sure what this means.  Is it that Rocq will verify itself??  Other readers may have the same question.
Performance isn't a good fit under the title "Trustworthy".  Would be better as a separate item